servo-build-dependencies/arm.sls

```diff
@@ -36,7 +36,7 @@ arm-dependencies:
 libs-{{ target.name }}:
   archive.extracted:
     - name: {{ common.servo_home }}/rootfs-trusty-{{ target.name }}/{{ target.version }}
-    - source: https://servo-rust.s3.amazonaws.com/ARM/{{ target.download_name }}/{{ target.version }}/{{ target.download_name }}-{{ target.version }}.tgz
+    - source: https://servo-rust.s3.amazonaws.com/ARM/{{ target.version }}/{{ target.download_name }}.tgz
```

Comment:
Please keep this URL the way it is - it's more verbose but it's more robust:

- Can update arm32 and arm64 independently
- If you download a bunch of tarballs from S3 , they'll have different names and will happily coexist on your filesystem